docs/contributor/04-10-workflows.md

```diff
@@ -75,3 +79,19 @@ The workflow:
 - Checks out code and sets up the cache
 - Sets up the Go environment
 - Invokes `make test`
+
+### E2E SAP BTP Manager Secret Customization Test
```

Comment:
And once again, see my first comment, which also applies to lines 85 and 97.

Reply:
suggestion applied.